frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -268,32 +270,36 @@ function initializeTracing(_annotation: APIAnnotation, tracing: HybridServerTrac
     // $FlowIssue[prop-missing] For some reason flow thinks the task property is missing, but it is not
     Store.dispatch(initializeAnnotationAction(annotation));
 
-    serverTracingAsVolumeTracingMaybe(tracing).map(volumeTracing => {
+    serverTracingAsVolumeTracings(serverTracings).map(volumeTracing => {
       ErrorHandling.assert(
         getSegmentationLayers(dataset).length > 0,
         messages["tracing.volume_missing_segmentation"],
       );
+      console.log("initializeVolumeTracingAction");
       Store.dispatch(initializeVolumeTracingAction(volumeTracing));
     });
 
-    serverTracingAsSkeletonTracingMaybe(tracing).map(skeletonTracing => {
+    const skeletonTracing = getNullableSkeletonTracing(serverTracings);
+    if (skeletonTracing != null) {
       // To generate a huge amount of dummy trees, use:
       // import generateDummyTrees from "./model/helpers/generate_dummy_trees";
       // tracing.trees = generateDummyTrees(1, 200000);
       Store.dispatch(initializeSkeletonTracingAction(skeletonTracing));
-    });
+    }
   }
 
   // Initialize 'flight', 'oblique' or 'orthogonal'/'volume' mode
   if (allowedModes.length === 0) {
     Toast.error(messages["tracing.no_allowed_mode"]);
   } else {
-    const isHybridTracing = tracing.skeleton != null && tracing.volume != null;
-    let maybeUrlViewMode = UrlManager.initialState.mode;
-    if (isHybridTracing && UrlManager.initialState.mode === constants.MODE_VOLUME) {
-      // Here we avoid going into volume mode in hybrid tracings.
-      maybeUrlViewMode = constants.MODE_PLANE_TRACING;
-    }
+    const maybeUrlViewMode = UrlManager.initialState.mode;
+    // todo: refactor MODE_VOLUME away or make this logic compatible
```

Comment:
@daniel-wer I think, I would simply remove MODE_VOLUME away. I skimmed the usages of the viewMode and `MODE_VOLUME` doesn't seem to have any relevance if I didn't miss anything. I only noticed that one can weirdly switch to the "volume" mode by pressing "m" repeatedly (the view mode dropdown doesn't even contain "volume" but when switching to that mode, it will show it anyway). Do you think there is a reason which speaks against removing that mode? Backwards-compatibility in the URL needs to be preserved obviously, but other than that, the removal seems like the a good idea to me.

Reply:
Agreed, seems like a leftover to me. :+1: